org.eclipse.xtext.testing/META-INF/MANIFEST.MF

```diff
@@ -27,7 +27,9 @@ Require-Bundle: org.eclipse.xtext;visibility:=reexport,
  org.eclipse.xtend.lib
 Import-Package: org.apache.log4j;version="1.2.15",
  org.apache.log4j.spi;version="1.2.15",
- org.junit.jupiter.api.extension;version="[5.0.0,6.0.0)";resolution:=optional
+ org.junit.jupiter.api.extension;version="[5.0.0,6.0.0)";resolution:=optional,
+ org.junit.platform.runner;resolution:=optional,
```

Comment:
no version range here?

Reply:
do we need to adapt targets in core, extras, eclipse, xtend & umbrella too?

Reply:
I don't think so since we use these only to run the Junit5 tests in testing.tests with a Junit4 runner